packages/miniflare/test/plugins/core/modules.spec.ts

```diff
@@ -278,6 +279,30 @@ You must manually define your modules when constructing Miniflare:
     at ${depPath}:2:8`
 	);
 });
+test.serial(
+	"Miniflare: collects modules outside of working directory",
+	async (t) => {
+		// https://github.com/cloudflare/workers-sdk/issues/4721
+		const tmp = await useTmp(t);
+		const child = path.join(tmp, "child");
+		await fs.mkdir(child);
+		await fs.writeFile(
+			path.join(tmp, "worker.mjs"),
+			'export default { fetch() { return new Response("body"); } }'
+		);
+		useCwd(t, child);
+
+		const mf = new Miniflare({
+			modules: true,
+			modulesRoot: "..",
+			scriptPath: "../worker.mjs",
```

Comment:
could we avoid the cwd change+resetting by choosing another tmp dir for the worker and using a relative path from the cwd to the other tmp dir?